Makefile

```diff
@@ -137,58 +160,106 @@ rm -rf $$TMP_DIR ;\
-	chmod +x $(OPM) ;\
-	}
-else
-OPM = $(shell which opm)
-endif
-endif
+# .PHONY: bundle
+# bundle: manifests kustomize ## Generate bundle manifests and metadata, then validate generated files.
+# 	operator-sdk generate kustomize manifests -q
+# 	cd config/manager && $(KUSTOMIZE) edit set image controller=$(IMG)
+# 	$(KUSTOMIZE) build config/manifests | operator-sdk generate bundle -q --overwrite --version $(VERSION) $(BUNDLE_METADATA_OPTS)
+# 	operator-sdk bundle validate ./bundle
 
-# A comma-separated list of bundle images (e.g. make catalog-build BUNDLE_IMGS=example.com/operator-bundle:v0.1.0,example.com/operator-bundle:v0.2.0).
-# These images MUST exist in a registry and be pull-able.
-BUNDLE_IMGS ?= $(BUNDLE_IMG)
+# .PHONY: bundle-build
+# bundle-build: ## Build the bundle image.
+# 	docker build -f bundle.Dockerfile -t $(BUNDLE_IMG) .
 
-# The image tag given to the resulting catalog image (e.g. make catalog-build CATALOG_IMG=example.com/operator-catalog:v0.2.0).
-CATALOG_IMG ?= $(IMAGE_TAG_BASE)-catalog:v$(VERSION)
+# .PHONY: bundle-push
+# bundle-push: ## Push the bundle image.
+# 	$(MAKE) docker-push IMG=$(BUNDLE_IMG)
+
+# .PHONY: opm
+# OPM = ./bin/opm
+# opm: ## Download opm locally if necessary.
+# ifeq (,$(wildcard $(OPM)))
+# ifeq (,$(shell which opm 2>/dev/null))
+# 	@{ \
+# 	set -e ;\
+# 	mkdir -p $(dir $(OPM)) ;\
+# 	OS=$(shell go env GOOS) && ARCH=$(shell go env GOARCH) && \
+# 	curl -sSLo $(OPM) https://github.com/operator-framework/operator-registry/releases/download/v1.15.1/$${OS}-$${ARCH}-opm ;\
+# 	chmod +x $(OPM) ;\
+# 	}
+# else
+# OPM = $(shell which opm)
+# endif
+# endif
+
+# # A comma-separated list of bundle images (e.g. make catalog-build BUNDLE_IMGS=example.com/operator-bundle:v0.1.0,example.com/operator-bundle:v0.2.0).
+# # These images MUST exist in a registry and be pull-able.
+# BUNDLE_IMGS ?= $(BUNDLE_IMG)
+
+# # The image tag given to the resulting catalog image (e.g. make catalog-build CATALOG_IMG=example.com/operator-catalog:v0.2.0).
+# CATALOG_IMG ?= $(IMAGE_TAG_BASE)-catalog:v$(VERSION)
+
+# # Set CATALOG_BASE_IMG to an existing catalog image tag to add $BUNDLE_IMGS to that image.
+# ifneq ($(origin CATALOG_BASE_IMG), undefined)
+# FROM_INDEX_OPT := --from-index $(CATALOG_BASE_IMG)
+# endif
+
+# # Build a catalog image by adding bundle images to an empty catalog using the operator package manager tool, 'opm'.
+# # This recipe invokes 'opm' in 'semver' bundle add mode. For more information on add modes, see:
+# # https://github.com/operator-framework/community-operators/blob/7f1438c/docs/packaging-operator.md#updating-your-existing-operator
+# .PHONY: catalog-build
+# catalog-build: opm ## Build a catalog image.
+# 	$(OPM) index add --container-tool docker --mode semver --tag $(CATALOG_IMG) --bundles $(BUNDLE_IMGS) $(FROM_INDEX_OPT)
+
+# # Push the catalog image.
+# .PHONY: catalog-push
+# catalog-push: ## Push a catalog image.
+# 	$(MAKE) docker-push IMG=$(CATALOG_IMG)
```

Comment:
Why is all this stuff commented out?

Reply:
this is the stuff generated by operator-sdk . I commented it out to check if we need this stuff in Makefile or we could remove it. There are some commands for creating `bundle` image,` catalog-source` etc. But for bundle stuff we use hack/sync_bundle anyway.